axis.go

```diff
@@ -295,15 +295,43 @@ func (a horizontalAxis) draw(c draw.Canvas) {
 }
 
 // GlyphBoxes returns the GlyphBoxes for the tick labels.
-func (a horizontalAxis) GlyphBoxes(*Plot) []GlyphBox {
-	var boxes []GlyphBox
-	for _, t := range a.Tick.Marker.Ticks(a.Min, a.Max) {
+func (a horizontalAxis) GlyphBoxes(p *Plot) []GlyphBox {
+	var (
+		boxes []GlyphBox
+		yoff  font.Length
+	)
+
+	if a.Label.Text != "" {
+		x := a.Norm(p.X.Max)
+		switch a.Label.Position {
+		case draw.PosCenter:
+			x = a.Norm(0.5*(p.X.Max-p.X.Min) + p.X.Min)
```

Comment:
`a.Norm(0.5*(p.X.Min+p.X.Max))`

Reply:
should I also apply this fix to `draw.Canvas.X/Y`?

Reply:
There are a few of them. Probably in another PR.

I'm trying to remember if someone preferred to express this as half-way plus the min rather than the mean of two points; I think the latter is clearer, but it does feel like there is a memory of that discussion rattling around in my head.

Reply:
Ok.
I'll file an issue when in front of a real keyboard.